Recrutify/Recrutify.Host/Extensions/MiddlewareExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Builder;
+
+namespace Recrutify.Host.Exceptions
```

Comment:
update namespace name

Reply:
this is right? namespace Recrutify.Host.Exceptions

Reply:
but your folder named Extensions, folder and namespace should be same